backend/app/tests/utils/test_db.py

```
@@ -328,7 +328,7 @@
     try:
         with engine.connect() as conn:
             result = conn.execute(text("SELECT 1")).scalar()
-            return result == 1
+            assert result == 1, "数据库连接测试未返回预期值"
     except Exception as e:
         logger.warning(f"Database connection test failed: {e}")
-        return False
+        assert False, f"数据库连接测试失败: {e}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)